packages/formStructure/components/FieldListColumn.tsx

```diff
@@ -5,13 +5,13 @@ type DefaultProps = Pick<TextInputProps, "key" | "id" | "inputLabel">;
 
 type Value<T> = T;
 
-interface RenderProps<T> {
+interface RenderProps<T, E> {
```

Comment:
should we add `React.FormEvent<HTMLInputElement>` here to make the transition easier?